code/__defines/radio.dm

```diff
@@ -124,3 +124,8 @@ var/list/DEPT_FREQS_ASSOC = list(
 #define RADIO_MULEBOT "radio_mulebot"
 #define RADIO_MAGNETS "radio_magnet"
 #define RADIO_ARRIVALS "radio_arrvl"
+
+#define JAMMER_BINARY -1 // can only be OFF or ALL
+#define JAMMER_OFF 0
+#define JAMMER_ALL 1 // affects ALL wireless streams
+#define JAMMER_SYNTHETIC 2 // affects only synthetic wireless connections (attack_ai)
```

Comment:
Perhaps it could be a bit field allowing for any combination of blocking? That would allow easy expansion in the future and some other "fun" like admins setting otherwise impossible combinations or such.

Reply:
for the moment, it doesn't need a bit field. anyone who wants to expand the system in the future, feel free to do so, it shouldn't be too difficult